tests/app/submitter/test_submitter.py

```diff
@@ -78,3 +78,30 @@ def test_when_fail_to_publish_message_then_returns_false(self):
 
             # Then
             self.assertFalse(published, 'send_message should fail to publish message')
+
+    def test_log_submitter_send_message(self):
+        # Given
+        with patch('app.submitter.submitter.BlockingConnection'):
+            # When
+            sent_message = LogSubmitter().send_message(message={}, queue='test_queue')
+
+            # Then
+            self.assertEqual(sent_message, True)
+
+    def test_get_submitter_settings_enabled(self):
+        settings.EQ_RABBITMQ_ENABLED = True
+        test = SubmitterFactory.get_submitter()
+        self.assertIsInstance(type(test), cls=object)
+
+    def test_get_submitter_settings_not_enabled(self):
+        settings.EQ_RABBITMQ_ENABLED = False
+        test = SubmitterFactory.get_submitter()
+        self.assertIsInstance(type(test), cls=object)
+
+    def test_submitter_send_message(self):
+        test = Submitter.send_message(self, message={}, queue='test_queue')
+        self.assertEqual(test, False)
+
+    def test_submitter_encrypt_message(self):
+        test = Submitter.encrypt_message(message={})
+        self.assertIsInstance(test, cls=str)
```

Comment:
You should test that the messages is actually being encrypted rather than just writing a test to touch lines.